FlashCardApp.sln

```diff
@@ -0,0 +1,16 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "FlashCardApp", "FlashCardApp\FlashCardApp.csproj", "{13A64592-DBDF-49F3-98C8-312CA73CCB65}"
```

Comment:
Could you please create a folder with the name "KimFom.Flashcards" and then create your project inside it? We want the parent folder to contain only folders. I know it's a bit confusing at first, but once you get the gist it will be a no-brainer :D

Reply:
Alrighty

Reply:
done